device/sycl/src/utils/thread_id.hpp

```diff
@@ -33,23 +33,33 @@ requires(DIMENSIONS >= 1 && DIMENSIONS <= 3) struct thread_id {
     /// @name Function(s) implementing @c traccc::device::concepts::thread_id1
     /// @{
 
-    inline auto getLocalThreadId() const {
-        return m_item.get_local_linear_id();
+    inline unsigned int getLocalThreadId() const {
```

Comment:
FYI you forgot this change in the Alpaka code.

Reply:
I didn't. 🤔 I was pondering about removing `auto` from there as well.

But I think you're right, we should do an explicit cast in that class as well. 🤔